### PR TITLE
Mustafa sh browsers02

### PR DESCRIPTION
**Pull request by Mustafa681**
add my name

### CODE REVIEW

**Review by FikretYilmaz — COMMENTED** [review]
I hope it works for merging

File: README.md [hunk 1]
```diff
@@ -78,5 +78,5 @@ This group of Ninja's consists of:
 * Amar Ibrahim
 * Claudia de Windt
 * Fikret Yilmaz
-
+* Mustafa Sh
```
Comment (FikretYilmaz) [inline README.md:81]: Same thing twice?

Comment (Mustafa681) [inline README.md:81]: it's not working no matter what i do

**Comment by Mustafa681** [comment]
w

**Pull request closed without merging**